cmd/nerdctl/container/container_stop_linux_test.go

```diff
@@ -159,3 +160,39 @@ func TestStopCreated(t *testing.T) {
 
 	base.Cmd("stop", tID).AssertOK()
 }
+
+func TestStopWithLongTimeoutAndSIGKILL(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	testContainerName := testutil.Identifier(t)
+	defer base.Cmd("rm", "-f", testContainerName).Run()
+
+	// Start a container that sleeps forever
+	base.Cmd("run", "-d", "--name", testContainerName, testutil.CommonImage, "sleep", "Inf").AssertOK()
+
+	// Stop the container with a 30-second timeout and SIGKILL
+	start := time.Now()
+	base.Cmd("stop", "--time=30", "--signal", "SIGKILL", testContainerName).AssertOK()
+	elapsed := time.Since(start)
+
+	// The container should be stopped almost immediately, well before the 30-second timeout
+	assert.Assert(t, elapsed < 30*time.Second, "Container wasn't stopped immediately with SIGKILL")
```

Comment:
Let's use a shorter timeout (say 5 sec) just in case so it won't block CI too long even when the test fails.